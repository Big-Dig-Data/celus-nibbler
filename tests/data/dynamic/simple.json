{
  "version": 1,
  "parser_name": "simple",
  "format_name": "simple_format",
  "platforms": [
    "Platform1",
    "Platform2"
  ],
  "dimensions": [
    "Dim1",
    "Dim2"
  ],
  "metrics_to_skip": [],
  "titles_to_skip": [],
  "platforms_to_skip": [],
  "metric_aliases": [],
  "dimension_aliases": [],
  "heuristics": {
    "conds": [
      {
        "pattern": "Title",
        "coord": {
          "row": 0,
          "col": 0
        },
        "kind": "regex"
      },
      {
        "pattern": "ISSN",
        "coord": {
          "row": 0,
          "col": 1
        },
        "kind": "regex"
      },
      {
        "pattern": "Platform",
        "coord": {
          "row": 0,
          "col": 2
        },
        "kind": "regex"
      },
      {
        "pattern": "YOP",
        "coord": {
          "row": 0,
          "col": 3
        },
        "kind": "regex"
      },
      {
        "pattern": "Metrics",
        "coord": {
          "row": 0,
          "col": 4
        },
        "kind": "regex"
      }
    ],
    "kind": "and"
  },
  "areas": [
    {
      "dates": {
        "direction": "down",
        "source": {
          "coord": {
            "row": 0,
            "col": 5
          },
          "direction": "right"
        }
      },
      "titles": {
        "source": {
          "coord": {
            "row": 1,
            "col": 0
          },
          "direction": "down"
        }
      },
      "metrics": {
        "source": {
          "coord": {
            "row": 1,
            "col": 4
          },
          "direction": "down"
        }
      },
      "platforms": {
        "source": {
          "coord": {
            "row": 1,
            "col": 2
          },
          "direction": "down"
        }
      },
      "title_ids": [
        {
          "name": "Print_ISSN",
          "source": {
            "coord": {
              "row": 1,
              "col": 1
            },
            "direction": "down"
          }
        }
      ],
      "dimensions": [
        {
          "name": "YOP",
          "source": {
            "coord": {
              "row": 1,
              "col": 3
            },
            "direction": "down"
          },
          "required": true
        }
      ],
      "name": "fixed"
    }
  ]
}
