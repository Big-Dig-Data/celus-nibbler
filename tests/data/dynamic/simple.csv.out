"2020-01-01","2020-01-31","T1","Sessions","Platform:P1|YOP:2015","Print_ISSN:1111-1111","1"
"2020-02-01","2020-02-29","T1","Sessions","Platform:P1|YOP:2015","Print_ISSN:1111-1111","2"
"2020-03-01","2020-03-31","T1","Sessions","Platform:P1|YOP:2015","Print_ISSN:1111-1111","3"
"2020-01-01","2020-01-31","T1","Denied","Platform:P1|YOP:2015","Print_ISSN:1111-1111","0"
"2020-02-01","2020-02-29","T1","Denied","Platform:P1|YOP:2015","Print_ISSN:1111-1111","1"
"2020-03-01","2020-03-31","T1","Denied","Platform:P1|YOP:2015","Print_ISSN:1111-1111","0"
"2020-01-01","2020-01-31","T2","Sessions","Platform:P1|YOP:2013","Print_ISSN:2222-2222","4"
"2020-02-01","2020-02-29","T2","Sessions","Platform:P1|YOP:2013","Print_ISSN:2222-2222","5"
"2020-03-01","2020-03-31","T2","Sessions","Platform:P1|YOP:2013","Print_ISSN:2222-2222","6"
"2020-01-01","2020-01-31","T2","Denied","Platform:P1|YOP:2013","Print_ISSN:2222-2222","3"
"2020-02-01","2020-02-29","T2","Denied","Platform:P1|YOP:2013","Print_ISSN:2222-2222","0"
"2020-03-01","2020-03-31","T2","Denied","Platform:P1|YOP:2013","Print_ISSN:2222-2222","2"
